### PR TITLE
Set up React Native scaffold (user app) and add test environment

### PR DESCRIPTION
**Pull request by emtalen**
### User Story
```
As a development team,
In order to be able to start adding features,
We would like to have a basic application structure and set up Cypress.
```
## Changes proposed in this pull request:
* Scaffolds React Native app
* Installing cypress and adding script for cypress open

## What I have learned working on this feature:
* How to scaffold a React Native app

## Packages/Gems added
- Cypress

### CODE REVIEW

**Comment by kfrostare** [comment]
Looks good to me /Kf

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (johanbounce) [inline cypress.json]: We need to figure out what to put here when we start using it

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (johanbounce) [inline cypress/fixtures/example.json]: delete file

**Review by johanbounce — COMMENTED** [review]
Fix changes

**Review by pgaunitz — APPROVED** [review]

File: cypress/fixtures/example.json (hunk 2 above)
Comment (kaylawoodbury) [reply cypress/fixtures/example.json]: removed

File: cypress.json (hunk 1 above)
Comment (kaylawoodbury) [reply cypress.json]: updated to add baseURL of localhost:19006

**Pull request merged**